src/webgpu/api/validation/queue/destroyed/buffer.spec.ts

```diff
@@ -0,0 +1,125 @@
+export const description = `
+Tests using a destroyed buffer on a queue.
+
+TODO:
+- test renderPass/renderBundle (setVertexBuffer, setIndexBuffer)
+- test renderPass (resolveQuerySet)
+- test renderPass/computePass (setBindGroup)
+`;
+
+import { makeTestGroup } from '../../../../../common/framework/test_group.js';
+import { ValidationTest } from '../../validation_test.js';
+
+export const g = makeTestGroup(ValidationTest);
+
+g.test('writeBuffer')
+  .desc(
+    `
+Tests that use a destroyed buffer in writeBuffer fails.
+- x= {destroyed, not destroyed (control case)}
+  `
+  )
+  .paramsSubcasesOnly(u => u.combine('buffer', ['valid', 'destroyed'] as const))
```

Comment:
What do you think about just using a bool and calling it `destroyed`? The rest of the test name is pretty self-explanatory already, and naming the param `buffer`/`texture` is kind of redundant/meaningless. (Here and in other applicable cases below where we only have 1 buffer/texture in the function so it is pretty clear that would be the one that is either destroyed or not.)

Reply:
done